Makefile

```diff
@@ -6,21 +6,19 @@ OBJCOPY = $(CROSS_COMPILE)objcopy
 OBJDUMP = $(CROSS_COMPILE)objdump
 READELF = $(CROSS_COMPILE)readelf
 
-OBJS	:= main.o benchmark_aux.o
+OBJS := main.o benchmark_aux.o
 
-CFLAGS += -Wall -I$(TEEC_EXPORT)/include\
-		-I./include -I$(TEEC_INTERNAL_INCLUDES)/include
+CFLAGS += -Wall -Wextra -Werror -I$(TEEC_EXPORT)/include -I$(TEEC_INTERNAL_INCLUDES)/include
 #Add/link other required libraries here
 LDADD += -lm -lteec -lpthread -L$(TEEC_EXPORT)/lib
-CROSS_COMPILE="$(HOST_CROSS_COMPILE)"
```

Comment:
You will need to update https://github.com/OP-TEE/build/blob/master/common.mk#L342: s/HOST_CROSS_COMPILE/CROSS_COMPILE/.